data/sound.appdata.xml.in

```diff
@@ -6,6 +6,11 @@
   <name>Sound Indicator</name>
   <summary>Adjust speaker and microphone volume or see Now Playing information and media controls</summary>
   <releases>
+    <release version="2.1.5" date="2019-12-15" urgency="medium">
+      <description>
+        <p>Corrected mpris text to Unkown Artist</p>
```

Comment:
typo ;) Also, `MPRIS` probably doesn't mean anything to most people so it might be better to say something like
```suggestion
        <p>Correctly show "Unknown Artist" when artist info isn't available</p>
```
Or something more user-facing

Reply:
Thanks. That's much better indeed.